analysis/plot.py

```diff
@@ -0,0 +1,62 @@
+"""Plot aggregated event counts.
```

Comment:
we could be more specific what the plot does here, and also give this file a more descriptive name (e.g., longitudinal_plot, trajectory_plot, event_count_plot)? 
```suggestion
"""Plot aggregated event counts over time.
```

Reply:
for me something that mentions time would make it clearer, but maybe it's so obvious that it's not needed for this ... and you're right, the terms longitudinal or trajectory might are field specific and maybe not the best example here

Reply:
so yes, I think you're right, different but not better ;)